cmd/keytransparency-server/Dockerfile

```diff
@@ -1,44 +1,13 @@
 FROM golang
 
-ENV DB_USER=test \
-    DB_PASSWORD=zaphod \
-    DB_DATABASE=test \
-    DB_HOST=db:3306
-
-ENV HOST=0.0.0.0 \
-    RPC_PORT=8080
-
-# TLS Certificate needs 0.0.0.0 to be in the SAN IP field.
-ENV VRF_PRIV=keytransparency/genfiles/vrf-key.pem \
-    TLS_KEY_PATH=keytransparency/genfiles/server.key \
-    TLS_CRT_PATH=keytransparency/genfiles/server.crt
-
-ENV MAP_ID=0 \
-    MAP_URL=""
-ENV LOG_ID=0 \
-    LOG_URL=localhost:8090
-
-ENV VERBOSITY=1
-
 ADD keytransparency/genfiles/* /kt/
 ADD ./keytransparency /go/src/github.com/google/keytransparency
 ADD ./trillian /go/src/github.com/google/trillian
 WORKDIR /go/src/github.com/google/keytransparency 
 
-RUN apt-get update && apt-get install -y libtool libltdl-dev
 RUN go get -tags="mysql" ./cmd/keytransparency-server
 
-ENTRYPOINT /go/bin/keytransparency-server \
```

Comment:
We need to set an entry point.  Otherwise, it's not possible to add commands via the docker commandline